package.json

```
@@ -10,8 +10,9 @@
   "author": "Kaholo Team",
   "license": "ISC",
   "dependencies": {
-    "package.json": "2.0.1",
-    "uuid": "8.3.2"
+    "kaholo-plugin-library": "Kaholo/kaholo-plugin-library",
```

Point to a specific release. If we merge new changes and these happen to be backwards compatible, then this line will break the plugin.
```suggestion
    "kaholo-plugin-library": "Kaholo/kaholo-plugin-library#v1.0.2",
```